modules/notice/notice.tsx

```diff
@@ -27,11 +34,11 @@ type Props = {
 	buttonDisabled?: boolean
 	header?: string
 	text?: string
-	style?: any
+	style?: StyleProp<ViewStyle>
 	spinner?: boolean
 	buttonText?: string
-	onPress?: () => any
-	textStyle?: any
+	onPress?: () => unknown
```

Comment:
```suggestion
	onPress?: () => void
```

(See my comment on the Animated PR)

Reply:
The comment makes sense, let's revert this.